tests/test_flow/test_persistence_fuzz.py

```diff
@@ -239,6 +275,11 @@ def run(self, n_iterations):
                     )
 
                 elif self._versioning_mode == 'assist':
+                    # No point in assisting when the updated entity changes per run.
```

Comment:
Are there no sensible assertions we can make here? It's possible to use `@changes_per_run` and assisted versioning in the same flow, right?

Reply:
Yes, you can use `assist` "mode" in the same flow that uses `@changes_per_run`, but this if conditional checks for the case when we did not update the version in assisted mode for a `changes_per_run` entity. 

It doesn't really make sense to use `@version` on the same entity as `@changes_per_run`.  I didn't want to go this far but maybe we should not even allow that. Wdyt?